js/id/ui/raw_tag_editor.js

```diff
@@ -186,7 +186,7 @@ iD.ui.RawTagEditor = function(context) {
             .on('change', keyChange);
 
         $items.select('input.value')
-            .value(function(d) { return d.value; })
+            .attr('value', function(d) { return d.value; })
```

Comment:
Okay, we need to make sure that is equivalent then.  Seems like that change might run a risk of breaking the functionality of the input.  Would be better to write a cucumber custom step that checks the value even if it can't be expressed in a simple css selector.